apps/ledger-live-mobile/src/components/RequiresCameraPermissions/useCameraPermissions.ts

```diff
@@ -0,0 +1,59 @@
+import { useCallback, useEffect, useMemo, useRef, useState } from "react";
+import { Camera } from "expo-camera";
+import { AppState, Linking } from "react-native";
+import { useIsMounted } from "../../helpers/useIsMounted";
+
+export default function useCameraPermissions() {
+  const [permission, requestPermission, checkPermission] =
+    Camera.useCameraPermissions();
```

Comment:
That's cheating ahah so much easier - i wish expo had an api for BLE and location permissions